packages/tablev2/Table.tsx

```diff
@@ -45,6 +45,11 @@ export type Column<A, B = string> = {
    * Text alignment for cells in the column
    */
   textAlign?: React.CSSProperties["textAlign"];
+
+  /**
+   * Text wrapping allows column content to wrap around instead of becoming cut off and truncated. The default is set to false to preserve truncation with backwards compatibility.
+   */
+  contentWrap?: boolean;
```

Comment:
because the default table behavior is wrap, and the existence of a boolean actually enables `nowrap`, you might consider inverting this prop. Making it `contentNoWrap` and defaulting it to `true` for backwards compatibility. either way, glad to have it.